commit a49725beede82687afd6603384f318afe9e60432
Author: Willy Tarreau <w@1wt.eu>
Date:   Fri Mar 29 17:35:32 2019 +0100

    BUILD: connection: fix naming of ip_v field
    
    AIX defines ip_v as ip_ff.ip_fv in netinet/ip.h using a macro, and
    unfortunately we do have a local variable with such a name and which
    uses the same header file. Let's rename the variable to ip_ver to fix
    this.
    
    (cherry picked from commit 0ca24aa028159874d77677076a835930de79ba8d)
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    (cherry picked from commit 0a5e7a9a39e14dccba4caa7df20cd3970f354078)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/src/connection.c b/src/connection.c
index 7403e8ae..f57ef60a 100644
--- a/src/connection.c
+++ b/src/connection.c
@@ -699,7 +699,7 @@ int conn_recv_netscaler_cip(struct connection *conn, int flag)
 {
 	char *line;
 	uint32_t hdr_len;
-	uint8_t ip_v;
+	uint8_t ip_ver;
 
 	/* we might have been called just after an asynchronous shutr */
 	if (conn->flags & CO_FL_SOCK_RD_SH)
@@ -765,9 +765,9 @@ int conn_recv_netscaler_cip(struct connection *conn, int flag)
 		goto missing;
 
 	/* Get IP version from the first four bits */
-	ip_v = (*line & 0xf0) >> 4;
+	ip_ver = (*line & 0xf0) >> 4;
 
-	if (ip_v == 4) {
+	if (ip_ver == 4) {
 		struct ip *hdr_ip4;
 		struct my_tcphdr *hdr_tcp;
 
@@ -797,7 +797,7 @@ int conn_recv_netscaler_cip(struct connection *conn, int flag)
 
 		conn->flags |= CO_FL_ADDR_FROM_SET | CO_FL_ADDR_TO_SET;
 	}
-	else if (ip_v == 6) {
+	else if (ip_ver == 6) {
 		struct ip6_hdr *hdr_ip6;
 		struct my_tcphdr *hdr_tcp;
 
